modules/microvm/virtualization/microvm/guivm.nix

```diff
@@ -108,6 +108,12 @@ let
                 pkgs.nm-launcher
                 pkgs.pamixer
               ]
+              # Packages for checking hardware acceleration
```

Comment:
Can you please check how much these add to the image size? And in any case, let's add them only to the `-debug`.